### DISCUSSION TITLE
Update Präsentation 14.11.2024

### DISCUSSION THREAD

**Discussion by lydiareiter** [plant2go/doc#9] [Announcements]
Heute war es wieder so weit gewesen. Wir haben Präsentiert

![](https://media1.tenor.com/m/BoieeZ9bvUUAAAAd/shock-eyes.gif)

<hr/>

Aber es ist besser angekommen als gedacht

![](https://media1.tenor.com/m/RdYowW9KtNMAAAAC/dancing-happy-dance.gif)

Um euch ein bisschen mehr zu motivieren. 
Das war das Feedback unserer Mitstudenten 

* Stockholder Map war ein bisschen klein
* finde euer ehrliches Fazit mega gut
* Farben in der PowerPoint sehr schön und einheitlich
* Fazit war super, Präsentation allgemein war auch übersichtlich und nachvollziehbar
* cool, dass auch freitextfragen integriert wurden :) bzw die "negativen" Feedback Punkte als Überleitung genommen habt zur weiteren Entwicklung
* cool, dass Ihr schon über Lösungen für Probleme nachgedacht habt
* Umfrage Antworten sehr schön übersichtlich
* Toll, dass ihr auch das negative Feedback einsehen könnt

<hr/>

Also ein Grund zu feiern

![](https://media1.tenor.com/m/Cc1qotSwPnIAAAAC/yee.gif)

**Comment by lenapopena** [comment]
![image](https://github.com/user-attachments/assets/c766b375-6518-4d26-a2dd-ff3ad8b71fcd)
